api/src/main/java/org/openmrs/module/insuranceclaims/util/ContextUtil.java

```diff
@@ -0,0 +1,21 @@
+package org.openmrs.module.insuranceclaims.util;
+
+
+import org.openmrs.api.context.Context;
+import org.openmrs.module.insuranceclaims.api.helper.DiagnosisHelper;
+import org.openmrs.module.insuranceclaims.api.helper.InsuranceClaimHelper;
+
+public class ContextUtil {
```

Comment:
should be final, however i do not see sense of this method. Just use autowired in different approach